sdmetrics/detection/tabular/base.py

```diff
@@ -54,22 +69,24 @@ def metrics(self, metadata, real_tables, fake_tables):
                 domain=(0.0, 1.0)
             )
 
-            # Parent-Child Table Detection
+    def _parent_child(self, metadata, real_tables, synthetic_tables):
```

Comment:
maybe name it  _combined_table_detection

Reply:
How about `_parent_child_detection`? "combined table" is a bit unclear/vague.